src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -76,9 +76,6 @@ import {
   vaEmployee,
   summaryOfEvidence,
   fullyDevelopedClaim,
-  unemployabilityStatus,
-  unemployabilityFormIntro,
-  workBehaviorChanges,
```

Comment:
I think workBehaviorChanges needs to stay in all-claims/config/form.js

Reply:
Great catch!!!